### PR TITLE
Vectorized PropagateInTube method

### PR DESCRIPTION
**Pull request by agarzaro**
Example of vectorized methods that works in the complex flow in vector mode.

### CODE REVIEW

File: examples/trackML/src/SimpleStepper.cxx [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-
 #include <management/GeoManager.h>
 #include <volumes/Tube.h>
 #include "Track.h"
 #include "HelixPropagator.h"
 #include "vectorFlow/SystemOfUnits.h"
+#include "Geant/ConstFieldHelixStepper.h"
+#include "Tracks_v.h"
+
+extern int gNkills;
```
Comment (agheata) [inline examples/trackML/src/SimpleStepper.cxx]: I do not really like passing configuration parameters as external globals. Rather make a data member, configuration object, ... that you can setup at initialisation time.

**Review by agheata — CHANGES_REQUESTED** [review]
Looks good, just fix the gNkills issue, then we can merge. Please enable also profiling for this example.

File: examples/trackML/src/SimpleStepper.cxx (hunk 1 above)
Comment (agarzaro) [reply examples/trackML/src/SimpleStepper.cxx]: Solved. Removed external global variable...

**Comment by agarzaro** [comment]
Issue solved & profiling enabled.

**Review by agheata — APPROVED** [review]

**Pull request merged**